### PR TITLE
apiary kafka metastore listener

### PR DESCRIPTION
**Pull request by rpoluri**

### CODE REVIEW

File: files/startup.sh [hunk 1]
```diff
@@ -106,6 +112,7 @@ fi
 sed "s/HIVE_METASTORE_LOG_LEVEL/$HIVE_METASTORE_LOG_LEVEL/" -i /etc/hive/conf/hive-log4j2.properties
 
 [[ ! -z $SNS_ARN ]] && export METASTORE_LISTENERS="${METASTORE_LISTENERS},com.expediagroup.apiary.extensions.events.metastore.listener.ApiarySnsListener"
+[[ ! -z $KAFKA_BOOTSTRAP_SERVERS ]] && export METASTORE_LISTENERS="${METASTORE_LISTENERS},com.expediagroup.apiary.extensions.events.metastore.kafka.listener.KafkaMetaStoreEventListener"
```
Comment (massdosage) [inline files/startup.sh]: I guess this depends on what we decide over in https://github.com/ExpediaGroup/apiary-data-lake/pull/134/files/1e2f79fbd87b4f9dc158a08d9b44575ed6dc193b..01939e1bab7747b57c58f323a3e322ebe994dacf#diff-c9ac8098c5ea9d3e6a9a596ff0c512a4 as to whether we introduce a separate variable to turn this on and off.

File: files/startup.sh [hunk 2]
```diff
@@ -121,6 +128,7 @@ sed "s/METASTORE_PRELISTENERS/${METASTORE_PRELISTENERS}/" -i /etc/hive/conf/hive
 
 export AUX_CLASSPATH="/usr/share/java/mariadb-connector-java.jar"
 [[ ! -z $SNS_ARN ]] && export AUX_CLASSPATH="$AUX_CLASSPATH:/usr/lib/apiary/apiary-metastore-listener-${APIARY_METASTORE_LISTENER_VERSION}-all.jar"
```
Comment (massdosage) [inline files/startup.sh]: Shouldn't we rather be using https://github.com/ExpediaGroup/apiary-data-lake/blob/master/variables.tf#L272 (`enable_metadata_events`) for this?

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.8.0] - 2020-02-06
+
```
Comment (massdosage) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
...
+### Changed
```
Comment (massdosage) [inline CHANGELOG.md]:
```suggestion
### Added
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
...
+- Add optional Apiary Kafka metastore listener, to send Hive metadata events to Kafka topic.
```
Comment (massdosage) [inline CHANGELOG.md]:
```suggestion
- Added an optional Apiary metastore listener which can be used to send Hive metadata events to a Kafka topic.
```

File: files/startup.sh [hunk 6]
```diff
@@ -71,6 +71,13 @@ then
     sed "s/ATLAS_CLUSTER_NAME/${ATLAS_CLUSTER_NAME}/g" -i /etc/hive/conf/atlas-application.properties
 fi
 
+#configure kafka metastore listener
+if [[ ! -z $KAFKA_BOOTSTRAP_SERVERS ]]; then
+    sed "s/KAFKA_BOOTSTRAP_SERVERS/$KAFKA_BOOTSTRAP_SERVERS/" -i /etc/hive/conf/hive-site.xml
+    sed "s/KAFKA_TOPIC_NAME/$KAFKA_TOPIC_NAME/" -i /etc/hive/conf/hive-site.xml
+    sed "s/KAFKA_CLIENT_ID/$HOSTNAME/" -i /etc/hive/conf/hive-site.xml
```
Comment (barnharts4) [inline files/startup.sh]: can we use `ECS_TASK_ID` when running in ECS?

**Pull request merged**